src/component/home/home.component.jsx

```diff
@@ -2,14 +2,21 @@ import React, { useEffect, useState, Fragment } from 'react';
 import HeaderTitle from '../_dumb/header-title';
 import { db } from '../data/firebase';
 import LectureSlider from '../lecture-slider';
+import { debounce } from 'lodash'
 
 const Home = () => {
+  const [pageY, setPageY] = useState(0)
   const [data, updateData] = useState({
     courseList: [],
     lectureList: [],
     sectionList: [],
   });
 
+  const handleScroll = debounce(() => {
```

Comment:
Just to make sure I understand: In this situation debounce is used for delaying the invocation of setPageY for 500s ?